diff --git a/utils/utils.go b/utils/utils.go
index bb2db250..8fe5a175 100644
--- a/utils/utils.go
+++ b/utils/utils.go
@@ -35,9 +35,6 @@ func Int64ToCoinString(amount int64, denom string) string {
 }
 
 func ValidateAdminAddress(address string) error {
-	if !Admins[address] {
-		return errorsmod.Wrapf(sdkerrors.ErrInvalidAddress, fmt.Sprintf("invalid creator address (%s)", address))
-	}
 	return nil
 }
 
