--- a/tx_template/Cargo.toml
+++ b/tx_template/Cargo.toml
@@ -10,10 +10,10 @@ version = "0.29.0"
 crate-type = ["cdylib"]
 
 [dependencies]
-namada_tx_prelude = {path = "../../tx_prelude"}
+namada_tx_prelude = {path = "../tx_prelude"}
 borsh = "1.2.0"
 wee_alloc = "0.4.5"
 getrandom = { version = "0.2", features = ["custom"] }
 
 [dev-dependencies]
-namada_tests = {path = "../../tests"}
+namada_tests = {path = "../tests"}
--- a/vp_template/Cargo.toml
+++ b/vp_template/Cargo.toml
@@ -10,10 +10,10 @@ version = "0.29.0"
 crate-type = ["cdylib"]
 
 [dependencies]
-namada_vp_prelude = {path = "../../vp_prelude"}
+namada_vp_prelude = {path = "../vp_prelude"}
 borsh = "1.2.0"
 wee_alloc = "0.4.5"
 getrandom = { version = "0.2", features = ["custom"] }
 
 [dev-dependencies]
-namada_tests = {path = "../../tests"}
+namada_tests = {path = "../tests"}
--- a/wasm_source/Cargo.toml
+++ b/wasm_source/Cargo.toml
@@ -40,8 +40,8 @@ vp_token = ["namada_vp_prelude"]
 vp_user = ["namada_vp_prelude", "once_cell"]
 
 [dependencies]
-namada_tx_prelude = {path = "../../tx_prelude", optional = true}
-namada_vp_prelude = {path = "../../vp_prelude", optional = true}
+namada_tx_prelude = {path = "../tx_prelude", optional = true}
+namada_vp_prelude = {path = "../vp_prelude", optional = true}
 borsh = "1.2.0"
 once_cell = {version = "1.8.0", optional = true}
 wee_alloc = "0.4.5"
@@ -51,11 +51,11 @@ masp_primitives = { git = "https://github.com/anoma/masp", tag = "v1.0.0", optio
 ripemd = "0.1"
 
 [dev-dependencies]
-namada = {path = "../../shared"}
-namada_tests = {path = "../../tests"}
-namada_test_utils = {path = "../../test_utils"}
-namada_tx_prelude = {path = "../../tx_prelude"}
-namada_vp_prelude = {path = "../../vp_prelude"}
+namada = {path = "../shared"}
+namada_tests = {path = "../tests"}
+namada_test_utils = {path = "../test_utils"}
+namada_tx_prelude = {path = "../tx_prelude"}
+namada_vp_prelude = {path = "../vp_prelude"}
 proptest = "1.4.0"
 test-log = {version = "0.2.14", default-features = false, features = ["trace"]}
 tracing = "0.1.30"
